webviz_subsurface/plugins/_reservoir_simulation_timeseries_onebyone.py

```diff
@@ -147,9 +147,9 @@ def __init__(
                 '"ensembles"'
             )
         if csvfile_smry and csvfile_parameters:
-            smry = read_csv(csvfile_smry)
-            parameters = read_csv(csvfile_parameters)
-            parameters["SENSTYPE"] = parameters.apply(
+            self.smry = read_csv(csvfile_smry)
+            self.parameters = read_csv(csvfile_parameters)
+            self.parameters["SENSTYPE"] = self.parameters.apply(
```

Comment:
Should we handle this some other way if we want to include the parameters in the data model sometime (instead of modifying the dataset)?